client-src/default/index.js

```diff
@@ -15,7 +15,7 @@ const status = {
   currentHash: '',
 };
 const options = {
-  hot: false,
+  hot: true,
```

Comment:
I think we can change it to false, because it is breaking `hot: false` logic, we run `.hot()` when hot enable, but if we use `hot: false` we doesn't run `.hot()` and default value now is `true`, so it can break logic for `hot: false`

Reply:
Good catch, I fixed it. The CI's error was because of this.